.github/scripts/pull_docker_images.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -e
+
+REPO=$(echo "docker.pkg.github.com/$GITHUB_REPOSITORY" | tr A-Z a-z)
+if [ -z "$IMAGES" ]; then
+    echo "Error: must put images to pull in \$IMAGES" >&2
+    exit 1
+fi
+
+for image in $IMAGES; do
+    PACKAGE="$REPO/dr_$image"
+    # Only pull the package if it already exists
+    (docker pull "$PACKAGE" && docker tag "$PACKAGE" "ccdlstaging/dr_$image") || true
```

Comment:
Hmm, I don't think I love pulling images and renaming them `ccdlstaging`, which is an actual docker repo we use. It seems like it'd be misleading downstream for someone who saw `ccdlstaging/dr_foreman`, they'd probably assume it came from that dockerhub repo. I think a better repo to standardize tags around is `localhost:5000`, cause that's where we run the local docker registry for end-to-end tests anyway. (If you do `docker push localhost:5000/dr_foreman` it'll push it to the local registry.

Reply:
The reason why I rename them to `ccdlstaging` is because all of our tests also build images under the `ccdlstaging` name, so this places the images where the tests expect them.

Reply:
Mmm yeah I figured that was why you did it. I think I regret doing that though. Basically I think that `ccdl` and `ccdlstaging` should be reserved for images that are pulled or pushed from those docker repos, but if we want a common name to reference them locally we shouldn't use `ccdlstaging` any more. The alternative to using `localhost:5000` would just be to set a `$DOCKERHUB_REPO` env var when we're running and reference that from all the tests. However, I think that would make more sense if this was where we actually built and pushed the images for deployments, but that happens on the deploy boxes everywhere.

If this is a big change we don't have to include it in this PR though.